tensorflow/python/ops/check_ops.py

```diff
@@ -1594,10 +1594,10 @@ def assert_shapes_v2(shapes, data=None, summarize=None, message=None,
 
   ```python
   tf.assert_shapes([
-    (x: ('N', 'Q')),
-    (y: ('N', 'D')),
-    (param: ('Q',)),
-    (scalar: ()),
+    (x, ('N', 'Q')),
```

Comment:
Though we accept tuples in v1 we only allow dictionaries in v2. I think the previous code needs to just use curly brackets instead.

Reply:
I agree.

Reply:
@jaingaurav I have updated this commit. Can you have a look at it?

Reply:
Shouldn't this be:

 ```
 tf.assert_shapes({
    x: ('N', 'Q'),
    y: ('N', 'D'),
    param: ('Q',),
    scalar: (),
  })
```

Reply:
Thanks for your prompt response.
That makes more clear and works on my side as well.

Reply:
@amitkumarj441: Seems like the code has changed since your changes. Could you please rebase and try again? Seems like there are 2 call sites to update in the example now.

Reply:
@jaingaurav I did rebase but seems CI is unhappy.